Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1839,10 +1839,23 @@
     io_registry_get_root_entry
     io_service_close))
 
+#if HAVE(MACH_HARDENED_EXCEPTIONS)
+        (with-filter (require-not (lockdown-mode))
+            (allow mach-message-send (kernel-mig-routine task_register_hardened_exception_handler)))
+            (allow mach-message-send (kernel-mig-routine thread_adopt_exception_handler)))
+#else
+        (with-filter (require-not (lockdown-mode))
+            (allow mach-message-send (kernel-mig-routine thread_set_exception_ports)))
+#end
+
```

Comment:
This part may be unnecessary, since this should be covered by `ernel-mig-routines-blocked-in-lockdown-mode-avoid-telemetry`.